runtime/gc_vlhgc/GlobalMarkingScheme.cpp

```diff
@@ -774,16 +777,33 @@ MM_GlobalMarkingScheme::scanPointerArrayObject(MM_EnvironmentVLHGC *env, J9Index
 	}
 }
 
+void
+MM_GlobalMarkingScheme::doSlot(MM_EnvironmentVLHGC *env, J9Object *fromObject, J9Object** slotPtr)
+{
+	markObject(env, *slotPtr);
+	rememberReferenceIfRequired(env, fromObject, *slotPtr);
+}
+
+#if JAVA_SPEC_VERSION >= 24
+void
+MM_GlobalMarkingScheme::doContinuationSlot(MM_EnvironmentVLHGC *env, J9Object *fromObject, J9Object** slotPtr, GC_ContinuationSlotIterator *continuationSlotIterator)
```

Comment:
minor but should be 'J9Object **slotPtr' (not 'J9Object** slotPtr')
correct in all spots that you touched it